lib/ansible/modules/windows/win_domain_group_membership.py

```diff
@@ -89,6 +90,15 @@
     members:
       - DESKTOP$
     state: present
+
+- name: Add a domain user/group from another Domain in the multi-domain forest to a domain group
+  win_domain_group_membership:
+    domain_server: DomainAAA.cloud
+    name: GroupinDomainAAA
+    members:
+      - "DomainBBB.cloud\\UserInDomainBBB"
```

Comment:
Remove the double quotes and this can just simply become `- DomainBBB.cloud\UserInDomainBBB`